org.OpenT2T.Sample.SuperPopular.Doorbell/org.OpenT2T.Sample.SuperPopular.Doorbell.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Doorbell">
+    
+   <!-- is the doorbell pressed? -->
+    <method name="isButtonPressed">
```

Comment:
Shouldn't this be a signal? Apps aren't going to constantly poll to check if a doorbell is pressed.

I know we don't yet have any example translators using signals. (I'm not even sure how well they work.)